modules/tinymce/src/core/test/ts/browser/textpatterns/FindInlinePatternTest.ts

```diff
@@ -30,12 +30,15 @@ describe('browser.tinymce.plugins.textpattern.FindInlinePatternTest', () => {
       { start: 'asap', replacement: 'as soon as possible' }
     ],
     base_url: '/project/tinymce/js/tinymce'
-  }, [ ListsPlugin, TextPatternPlugin ]);
+  }, [ ListsPlugin ]);
 
-  const inlinePatterns = Thunk.cached(() => Options.getPatternSet(hook.editor()).inlinePatterns);
+  const inlinePatterns = Thunk.cached(() => {
+    const rawPatterns = Options.getTextPatterns(hook.editor()) as RawPattern[];
```

Comment:
Why is the cast needed here as it shouldn't be?

Reply:
This is the test I mentioned in the call the other day. But true, this shouldn't be needed with the type change from earlier today...

Reply:
Fixed

Reply:
Ahh right I forgot about that sorry.